### ISSUE TITLE
More recent GEOmetadb files?

### ISSUE THREAD

**Issue by tgbrooks** [zhujack/GEOmetadb#3]
The file that GEOmetadb downloads (https://gbnci.cancer.gov/geo/GEOmetadb.sqlite.gz) seems to have last been updated on November 2021. Is there any way to obtain a more recent version? Thanks.

**Comment by Yunuuuu** [comment]
Is there a way to check the date?

**Comment by tgbrooks** [comment]
I just looked at the date of the most recent data in the file. Checking today, it looks like the most recent data is from 2023-09-01 so it has been updated pretty recently:

``` SQL
> SELECT max(submission_date) FROM gse;
2023-09-01
```

A faster way to check (without downloading the whole file) is to run:

``` bash
 curl -s -v -X HEAD https://gbnci.cancer.gov/geo/GEOmetadb.sqlite.gz 2>&1 | grep '^< Last-Modified:'
```

This gives me Tue, 05 Sep 2023, which is 4 days more recent than the actual most recent data.


Guess I'll close this issue, though it would be nice if there were an official response about the update schedule.
